src/jdom/service.ts

```diff
@@ -488,6 +492,18 @@ export class JDService extends JDNode {
             } else if (pkt.isEvent) {
                 const ev = this.event(pkt.eventCode)
                 if (ev) ev.processEvent(pkt)
+            } else if (pkt.serviceCommand === SystemCmd.CommandNotImplemented) {
+                const [serviceCommand, packetCrc] =
+                    pkt.jdunpack<[number, number]>("u16 16")
+                console.debug(`not impl`, { serviceCommand, packetCrc })
+                if (
```

Comment:
@mmoskal is this where you want to check for not-implemented?

Reply:
yup!